app/views/partials/messageContent.scala.html

```diff
@@ -26,18 +26,21 @@
 @(message: MessageView)(implicit messages: Messages, requestHeader: RequestHeader)
 @import message._
 <p>
-<span class="govuk-caption-m">
+<span aria-hidden="true" class="govuk-caption-m">
     <span class="govuk-caption-m-!-govuk-body govuk-!-font-weight-bold">@sender sent</span>  @sentText
 </span>
+<span class="govuk-visually-hidden">@sender sent  @sentText </span>
```

Comment:
I think this is why the ConversationMessagesPartialISpec test is failing. The page now has double the number of dates (although half of them are hidden from view).

Reply:
updated the IT test